tools/wpt/expectation.json

```diff
@@ -2165,32 +2165,26 @@
     "url-setters.any.html": [
       "URL: Setting <http://example.net/path>.hostname = 'example.com:8080' : delimiter invalidates entire value",
       "URL: Setting <http://example.net:8080/path>.hostname = 'example.com:' : delimiter invalidates entire value",
-      "URL: Setting <non-spec:/.//p>.hostname = 'h' Drop /. from path",
-      "URL: Setting <non-spec:/.//p>.hostname = ''",
       "URL: Setting <foo://somehost/some/path>.pathname = '' Non-special URLs can have their paths erased",
       "URL: Setting <foo:///some/path>.pathname = '' Non-special URLs with an empty host can have their paths erased",
       "URL: Setting <file://monkey/>.pathname = '\\\\' File URLs and (back)slashes",
       "URL: Setting <file:///unicorn>.pathname = '//\\/' File URLs and (back)slashes",
       "URL: Setting <file:///unicorn>.pathname = '//monkey/..//' File URLs and (back)slashes",
       "URL: Setting <non-spec:/>.pathname = '/.//p' Serialize /. in path",
       "URL: Setting <non-spec:/>.pathname = '/..//p'",
-      "URL: Setting <non-spec:/>.pathname = '//p'",
-      "URL: Setting <non-spec:/.//>.pathname = 'p' Drop /. from path"
+      "URL: Setting <non-spec:/>.pathname = '//p'"
     ],
     "url-setters.any.worker.html": [
       "URL: Setting <http://example.net/path>.hostname = 'example.com:8080' : delimiter invalidates entire value",
       "URL: Setting <http://example.net:8080/path>.hostname = 'example.com:' : delimiter invalidates entire value",
-      "URL: Setting <non-spec:/.//p>.hostname = 'h' Drop /. from path",
-      "URL: Setting <non-spec:/.//p>.hostname = ''",
       "URL: Setting <foo://somehost/some/path>.pathname = '' Non-special URLs can have their paths erased",
       "URL: Setting <foo:///some/path>.pathname = '' Non-special URLs with an empty host can have their paths erased",
       "URL: Setting <file://monkey/>.pathname = '\\\\' File URLs and (back)slashes",
       "URL: Setting <file:///unicorn>.pathname = '//\\/' File URLs and (back)slashes",
       "URL: Setting <file:///unicorn>.pathname = '//monkey/..//' File URLs and (back)slashes",
       "URL: Setting <non-spec:/>.pathname = '/.//p' Serialize /. in path",
       "URL: Setting <non-spec:/>.pathname = '/..//p'",
-      "URL: Setting <non-spec:/>.pathname = '//p'",
-      "URL: Setting <non-spec:/.//>.pathname = 'p' Drop /. from path"
```

Comment:
Why are these tests getting enabled? Which changes caused it? Could we do that in a separate PR?